core/src/main/scala/cats/data/package.scala

```diff
@@ -29,6 +29,7 @@ package object data extends ScalaVersionSpecificPackage {
   type IorNes[B, +A] = Ior[NonEmptySet[B], A]
   type EitherNel[+E, +A] = Either[NonEmptyList[E], A]
   type EitherNec[+E, +A] = Either[NonEmptyChain[E], A]
+  type EitherNev[+E, +A] = Either[NonEmptyVector[E], A]
```

Comment:
for consistency should we add `IorNev` and `ValidatedNev`?

Reply:
good idea! sure!